ansible/tasks/setup-gotrue.yml

```
@@ -30,6 +30,13 @@
     owner: gotrue
     mode: 0775
 
+- name: gotrue - create /etc/auth.d
+  file:
+    path: /etc/auth.d
+    state: directory
+    owner: gotrue
+    mode: 0755
```

config dir should be set to 0750 as per [this comment](https://github.com/supabase/supabase-admin-api/pull/225#discussion_r1765073702) right?
```suggestion
    mode: 0750
```

---

I think that would be great if we could tighten up permission some, I've left a comment explaining why I don't think we can do that just yet: https://github.com/supabase/supabase-admin-api/pull/225#issuecomment-2369255240